src/main/java/com/epam/prd/jjop/helpers/unmodifiableList_vs_List_of/Main.java

```diff
@@ -0,0 +1,104 @@
+package com.epam.prd.jjop.helpers.unmodifiableList_vs_List_of;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The purpose of this class is to show the difference between:
+ * => .unmodifiableList()
+ * => List.of()
```

Comment:
```suggestion
 * <ul>
 *   <li>{@link java.util.Collections#unmodifiableList(List<? extends T>)</li>
 *   <li>java.util.List#of()</li>
 * </ul>
```

You may link directly to proper classes and their members via `{@link ...}` in javadocs. I may slightly mistake the syntax but it is close to suggestion above.